doc/clustering.md

```diff
@@ -634,7 +634,7 @@ This is done by using the `@` prefix when using `--target`.
 
 An example:
 ```bash
-lxc launch images:ubuntu/20.04 cluster:ubuntu --target=@gpu
+lxc launch images:ubuntu/22.04 cluster:ubuntu --target=@gpu
```

Comment:
Should this be `ubuntu:22.04`?

Reply:
Both work but in doc/clustering.md, the official Ubuntu image is used already so this is not consistent. Not sure which we should align to.

Reply:
It looks like we are using `ubuntu:...` most other places, except for when we are using the `/cloud` variants.

Reply:
Same